tests/test_branch_probing_lp.py

```diff
@@ -42,6 +42,7 @@ def branchexeclp(self, allowaddcons):
 
 def test_branching():
     m = Model()
+    m.setHeuristics(SCIP_PARAMSETTING.OFF)
     m.setIntParam("presolving/maxrounds", 0)
     #m.setLongintParam("lp/rootiterlim", 3)
     m.setRealParam("limits/time", 60)
```

Comment:
Since the code only works when it's called the first time, how about we make this node limit instead of 2? that should be system agnostic and would probably be faster. 
```suggestion
    m.setLongintParam("limits/nodes", 2)
```

Reply:
Yes, it sounds good. I made a similar change but with "limits/node" as 1 instead of 2. This parameter corresponds to the number of processed nodes. So the node limit of 1 works as intended.